src/Common/tests/System/Diagnostics/RemoteExecutorConsoleApp/project.json

```diff
@@ -1,13 +1,13 @@
 {
   "dependencies": {
-    "Microsoft.NETCore.Platforms": "1.0.1-rc3-23910",
-    "System.Console": "4.0.0-rc3-23910",
-    "System.IO": "4.0.10",
-    "System.Reflection": "4.0.0",
-    "System.Runtime": "4.0.20",
-    "System.Threading.Tasks": "4.0.10"
+    "Microsoft.NETCore.Platforms": "1.0.1-rc3-23918",
```

Comment:
Yes, it contains the lineup that brings in the implementations, also the runtime graph.  It should be used whenever resolving against a concrete TFM.